dist/completer.js.map

```diff
@@ -1 +1 @@
A,MAAMA,CAACA,SAASA,IAAIA,IAAIA,CAACA,UAAUA,CAACA,gBAAgBA;wBACpEA,MAAMA,EAAEA,iBAAiBA,GAAGA,UAAUA,GAAGA,GAAGA;wBAC5CA,IAAIA,EAAEA,SAASA;qBAChBA,CAACA;oBACFA,IAAIA,IAAIA,GAAGA,IAAIA,CAACA;oBAChBA,MAAMA,CAACA,IAAIA,CAACA,UAAUA,CAACA,sBAAsBA,CAACA,MAAMA,EAAEA,EAAEA,KAAKA,EAAEA,IAAIA,CAACA,OAAOA,CAACA,SAASA,EAAEA,EAAEA,CAACA,CAACA,IAAIA,CAACA,UAAAA,QAAQA;wBACtGA,IAAIA,CAACA,gBAAgBA,CAACA,UAAUA,CAACA,GAAGA,QAAQA,CAACA,UAAUA,CAACA;wBACxDA,MAAMA,CAACA,QAAQA,CAACA,UAAUA,CAACA;oBAC7BA,CAACA,CAACA,CAACA;gBACLA,CAACA;gBAEDH,2DAAsBA,GAAtBA,UAAuBA,KAAKA,EAAEA,IAAIA;oBAChCI,MAAMA,CAACA,KAAKA,CAACA,GAAGA,CAACA,UAAAA,IAAIA;wBACnBA,MAAMA,CAACA;4BACLA,OAAOA,EAAEA,IAAIA;4BACbA,KAAKA,EAAEA,IAAIA;4BACXA,IAAIA,EAAEA,IAAIA;4BACVA,KAAKA,EAAEA,MAAMA,CAACA,SAASA;yBACxBA,CAACA;oBACJA,CAACA,CAACA,CAACA;gBACLA,CAACA;gBAEDJ,8CAASA,GAATA,UAAUA,OAAOA,EAAEA,GAAGA,EAAEA,MAAMA,EAAEA,MAAMA,EAAEA,KAAKA,EAAEA,KAAKA;oBAClDK,IAAIA,MAAMA,EAAEA,GAAGA,CAACA;oBAChBA,GAAGA,CAACA,CAACA,GAAGA,CAACA,CAACA,GAAGA,GAAGA,EAAEA,CAACA,IAAIA,CAACA,EAAEA,CAACA,EAAEA,EAAEA,CAACA;wBAC9BA,MAAMA,GAAGA,OAAOA,CAACA,SAASA,CAACA,CAACA,CAACA,CAACA;wBAC9BA,EAAEA,CAACA,CAACA,CAACA,KAAKA,GAAGA,CAACA,CAACA,CAACA;4BACdA,IAAIA,CAACA,GAAGA,CAACA,CAACA;4BACVA,GAAGA,CAACA,CAACA,GAAGA,GAAGA,CAACA,EAAEA,GAAGA,GAAGA,MAAMA,CAACA,MAAMA,EAAEA,GAAGA,EAAEA,EAAEA,CAACA;gCACzCA,CAACA,IAAIA,MAAMA,CAACA,GAAGA,CAACA,CAACA,KAAKA,CAACA,MAAMA,CAACA;gCAC9BA,EAAEA,CAACA,CAACA,CAACA,IAAIA,MAAMA,CAACA,CAACA,CAACA;oCAChBA,KAAKA,CAACA;gCACRA,CAACA;4BACHA,CAACA;wBACHA,CAACA;wBAACA,IAAIA,CAACA,CAACA;4BACNA,GAAGA,GAAGA,MAAMA,CAACA,MAAMA,GAAGA,CAACA,CAACA;wBAC1BA,CAACA;wBAEDA,GAAGA,CAACA,CAACA,EAAEA,GAAGA,IAAIA,CAACA,EAAEA,GAAGA,EAAEA,EAAEA,CAACA;4BACvBA,EAAEA,CAACA,CAACA,MAAMA,CAACA,GAAGA,CAACA,CAACA,IAAIA,KAAKA,KAAKA,CAACA,CAACA,CAACA;gCAC/BA,MAAMA,CAACA,IAAIA,CAACA;4BACdA,CAACA;4BAEDA,EAAEA,CAACA,CAACA,MAAMA,CAACA,GAAGA,CAACA,CAACA,IAAIA,KAAKA,MAAMA;mCAC1BA,CAACA,CAACA,KAAKA,IAAIA,MAAMA,CAACA,GAAGA,CAACA,CAACA,KAAKA,KAAKA,KAAKA,CAACA,CAACA,CAACA,CAACA;gCAC7CA,MAAMA,CAACA,GAAGA,CAACA,CAACA,GAAGA,GAAGA,CAACA,CAACA;gCACpBA,MAAMA,CAACA,GAAGA,CAACA,CAACA,KAAKA,GAAGA,GAAGA,CAACA;gCACxBA,MAAMA,CAACA,MAAMA,CAACA,GAAGA,CAACA,CAACA;4BACrBA,CAACA;wBACHA,CAACA;oBACHA,CAACA;oBAEDA,MAAMA,CAACA,IAAIA,CAACA;gBACdA,CAACA;gBAEDL,kDAAaA,GAAbA,UAAcA,GAAGA,EAAEA,MAAMA,EAAEA,IAAIA;oBAA/BM,iBAcCA;oBAbCA,gBAACA,CAACA,MAAMA,CAACA,GAAGA,EAAEA,UAACA,GAAGA,EAAEA,GAAGA;wBACrBA,EAAEA,CAACA,CAACA,gBAACA,CAACA,QAAQA,CAACA,GAAGA,CAACA,CAACA,CAACA,CAACA;4BACpBA,KAAIA,CAACA,aAAaA,CAACA,GAAGA,EAAEA,MAAMA,GAAGA,GAAGA,GAAGA,GAAGA,EAAEA,IAAIA,CAACA,CAACA;wBACpDA,CAACA;wBAACA,IAAIA,CAACA,EAAEA,CAACA,CAACA,gBAACA,CAACA,OAAOA,CAACA,GAAGA,CAACA,CAACA,CAACA,CAACA;wBAE5BA,CAACA;wBAACA,IAAIA,CAACA,EAAEA,CAACA,CAACA,GAAGA,KAAKA,QAAQA,CAACA,CAACA,CAACA;wBAE9BA,CAACA;wBAACA,IAAIA,CAACA,CAACA;4BACNA,IAAIA,CAACA,IAAIA,CAACA,MAAMA,GAAGA,GAAGA,CAACA,CAACA;wBAC1BA,CAACA;oBACHA,CAACA,CAACA,CAACA;oBAEHA,MAAMA,CAACA,IAAIA,CAACA;gBACdA,CAACA;gBACHN,iCAACA;YAADA,CAACA,AA1JD,IA0JC;YA1JD,gDA0JC,CAAA"}
```

Comment:
Should this file be in source control?

Reply:
I believe that grafana plugins are expected to be compiled and fully commited to the repo.

See: http://docs.grafana.org/plugins/developing/development/#buildscript for explanation why.